wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java

```diff
@@ -79,13 +97,17 @@ public void writeData(String dvPath){
 				dc.units=ddsfl.getUnits().toUpperCase();
 				dc.values = values;
 				//boolean storeFlags = false;
-				dc.setName("/test/"+DssOperation.getTSName(dvAliasName)+"/"+ddsfl.getKind()+"//"+ddsfl.getTimeStep()+"/test/");
+				outPath = "/test/"+DssOperation.getTSName(dvAliasName)+
+						"/"+ddsfl.getKind()+"//"+ddsfl.getTimeStep()+"/test/";
+				dc.setName(outPath);
 				Calendar startCalendar=Calendar.getInstance();
 				startCalendar.setTime(ddsfl.getStartTime());
-				dc.setStartTime(new HecTime(startCalendar));
+				HecTime startHecTime = new HecTime(startCalendar);
+				dc.setStartTime(startHecTime);
 				dc.setStoreAsDoubles(true);
 				try {
 					dvDss.put(dc);
+					dvDss.close();
```

Comment:
DSS close should always be in a finally block so that a lock isn't held between sessions